new-integration-tests/src/test/resources/python-scripts/introspect_version_script.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+import sys, traceback
+
+def change_server_count():
+  try:
+    connect_to_adminserver()
+    edit()
+    startEdit()
+    cd('/Clusters/' + cluster_name + '/DynamicServers/' + cluster_name)
+    cmo.setDynamicClusterSize(6)
+    cmo.setMaxDynamicClusterSize(6)
+    cmo.setMinDynamicClusterSize(2)
+    save()
+    activate()
+    disconnect()
+  except NameError, e:
+    print('Apparently properties not set.')
+    print('Please check the property: ', sys.exc_info()[0], sys.exc_info()[1])    
+    exit(exitcode=1)
+  except:
+    print 'Deployment failed'
```

Comment:
There is no Deployment is happening.  May be cut-n-paste error

Reply:
Fixed